src/FixedDataTable.js

```diff
@@ -557,6 +618,8 @@ class FixedDataTable extends React.Component {
     const { scrollEnabledX, scrollEnabledY } = scrollbarsVisible(this.props);
     const onColumnReorder = onColumnReorderEndCallback ? this._onColumnReorder : null;
 
+    this._scrollbarYHeight = scrollbarYHeight;
```

Comment:
@wcjordan , as per this [comment](https://github.com/schrodinger/fixed-data-table-2/pull/207#issuecomment-341871923), should we change how the private variable `_scrollbarYHeight` is defined and used?

Now we get `scrollbarYHeight `from the selector `tableHeights`. We can move extracting this value on the event handler, but I wasn't sure if that's a better way than this (cause tableHeights is recalculated). What do you think?

Reply:
Then that sounds good. Have removed `_scrollbarYHeight`, and used it directly at the event handler.